src/Tribe/Admin/Settings.php

```diff
@@ -46,6 +46,18 @@ public function get_url( array $args = [] ) {
 
 		$wp_url = is_network_admin() ? network_admin_url( 'settings.php' ) : admin_url( 'edit.php' );
 
+		if ( $args['anchor'] ?? false ) {
+			// Append hash character if needed.
```

Comment:
One word substitution I missed before:

```suggestion
			// Prepend hash character if needed.
```